snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/SnomedEditingContext.java

```diff
@@ -791,8 +791,11 @@ public void preCommit() {
 				}
 			}
 		}
+		
+		new EffectiveTimeRestorer(getBranch()).restoreEffectiveTimes(transaction.getChangeSetData().getChangedObjects().stream()
```

Comment:
Why not instantiate the restorer in the constructor and just pass the branch to the method?